tests/src/Kernel/PriceTwigExtensionTest.php

```diff
@@ -0,0 +1,60 @@
+<?php
+
+namespace Drupal\Tests\commerce\Kernel;
+
+use Drupal\KernelTests\KernelTestBase;
+
+/**
+ * Tests the price twig filter.
+ *
+ * @group commerce
+ */
+class PriceTwigExtensionTest extends KernelTestBase {
+
+  /**
+   * Load in USD currency.
+   */
+  public function setUp() {
+    parent::setUp();
+    $this->container->get('commerce_price.currency_importer')->import('USD');
+  }
+
+  public static $modules = [
```

Comment:
$modules needs to be defined above any methods, and needs to have a docblock like in other tests.